code/test/CoreTemplateStudio.Core.Test/Extensions/StringExtensionsTests.cs

```diff
@@ -10,11 +10,55 @@ namespace Microsoft.Templates.Core.Test.Extensions
     [Trait("ExecutionSet", "Minimum")]
     public class StringExtensionsTests
     {
+        [Theory]
+        [InlineData("string")]
+        [InlineData("string23TEST")]
+        public void ObfuscateSHA_ShouldNotBeEmpty(string value)
+        {
+            var actual = value.ObfuscateSHA();
```

Comment:
I'm not clear what the method being tested here should do, and these tests don't help.
Does  the `ObfuscateSHA` method obfuscate a string or does it calculate a hash (as implied by the "SHA" part of the name)?
Hashing and obfuscating are not the same thing. Anyone encountering the method in the source code is likely to ask what the method does. The name isn't clear and the tests don't make it clear what the method is supposed to do.
I think some additional documentation (in the code) and possibly a renaming of the method are in order.